lib/grunt/utils.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
       done();
       return;
     }
-    var p = spawn('node', ['node_modules/protractor/bin/webdriver-manager', 'update']);
+    var p = spawn('node', [require.resolve('protractor/bin/webdriver-manager'), 'update']);
```

Comment:
`webdriver-manager` is a separate package (and Protractor [just uses it](https://github.com/angular/protractor/blob/79184ff13e9370e0803b70000f67b601934866ec/bin/webdriver-manager)) so we could switch as well.